presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1876,8 +1877,8 @@ public Void visitVariableReference(VariableReferenceExpression variable, Immutab
                                 hivePartitionResult.getEnforcedConstraint(),
                                 hiveBucketHandle,
                                 hivePartitionResult.getBucketFilter(),
-                                createTableLayoutString(session, handle.getSchemaTableName(), hivePartitionResult.getBucketHandle(), TRUE_CONSTANT, domainPredicate))),
-                hivePartitionResult.getUnenforcedConstraint()));
+                                createTableLayoutString(session, handle.getSchemaTableName(), hivePartitionResult.getBucketHandle(), TRUE_CONSTANT, domainPredicate),
+                                isPushdownFilterSupported(session, tableHandle))), hivePartitionResult.getUnenforcedConstraint()));
```

Comment:
- this should always be false 
- `hivePartitionResult.getUnenforcedConstraint()` needs to stay on a separate line